src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json

```diff
@@ -2,6 +2,7 @@
   "persons": [ {
     "name": "Hans Muster",
     "email": "invalid@email!3e",
-    "module": "CS1010J"
+    "module": "CS1010J",
+    "assignments": [ ]
```

Comment:
Perhaps we can add an assignment that is of the form [Assignment 2||] which is invalid